kpi/renderers.py

```diff
@@ -21,6 +24,33 @@ class AssetJsonRenderer(renderers.JSONRenderer):
     format = 'json'
 
 
+class MediaFileRenderer(renderers.BaseRenderer):
+    media_type = '*/*'
+    format = None
+    charset = None
+    render_style = 'binary'
+
+    def render(self, data, accepted_media_type=None, renderer_context=None):
+        return data
+
+
+class MP3ConversionRenderer(MediaFileRenderer):
+    media_type = 'audio/mpeg'
+    format = 'mp3'
+
+    def render(self, data, accepted_media_type=None, renderer_context=None):
+        try:
+            audio = AudioSegment.from_file(BytesIO(data))
+        except:
```

Comment:
Try to not catch too broad exception.
What kind of exception are you expecting?